src/Directory.Build.props

```
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DefaultNetCoreApp>net6.0</DefaultNetCoreApp>
+    <DefaultNetCoreApp>net8.0</DefaultNetCoreApp>
```

```suggestion
    <DefaultNetCoreApp>net8.0;net9.0</DefaultNetCoreApp>
```
Why we don't target `net9` too?

---

There's no need to target `net9.0` in a library when it targets `net8.0`, unless you need to use APIs that were introduced in .NET 9. Adding multiple targets just increases the NuGet package size for no benefit.

We follow the same approach in .NET Aspire. Most libraries target `net8.0`, and only if the library needs to use a .NET 9 API (right now only EF Core libraries need this) does it multi-target to both.

The tests, on the other hand, are executable. Multi-targeting them means that the tests will execute on both .NET 8 and .NET 9.